packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -162,37 +168,118 @@ function Section(props) {
     onSectionSelect(id);
   }, [id, onSectionSelect]);
 
+  const sectionHeader = (
+    <tbody>
+      {/* Render section rows */}
+      {!isHidden && (
+      <tr
+        aria-rowindex={sectionRowIndex}
+        className={cx('header-row', theme.className)}
+        data-section-id={id}
+      >
+        <th
+          id={`${tableId}-${id}`}
+          className={cx('header-cell')}
+          align="left"
+          colSpan={displayedColumns.length}
+          role="columnheader"
+          scope="col"
+          tabIndex={isGridContext && !hasSectionButton ? -1 : undefined}
+        >
+          <SectionHeader
+            className={cx('section-header')}
+            text={text}
+            isOpen={hasSectionButton ? !isCollapsed : undefined}
+            onClick={hasSectionButton ? handleClick : undefined}
+            isTitleSticky
+            boundedWidth={boundingWidth}
+          />
+        </th>
+      </tr>
+      )}
+    </tbody>
+  );
+
+  const rowProps = {
+    displayedColumns,
+    firstRowId,
+    height: rowHeight,
+    isTableStriped,
+    lastRowId,
+    onCellSelect,
+    rowHeaderIndex,
+    rowMinimumHeight,
+    rowSelectionMode,
+    sectionId: !isHidden ? id : undefined,
+    tableId,
+  };
+
+  if (subsections) {
+    return (
+      <>
+        {sectionHeader}
+        {!isCollapsed && subsections.map((subsection) => (
+          <>
+            <tbody className={cx('subsection', {
+              collapsed: isCollapsed,
+              collapsible: isCollapsible,
+            }, theme.className)}
+            >
+
+              <tr
+                aria-rowindex={subsection.subSectionRowIndex}
+                className={cx('header-row', theme.className)}
+                data-section-id={id}
+              >
+                <th
+                  id={`${tableId}-${id}-${subsection.id}`}
+                  className={cx('header-cell')}
+                  align="left"
+                  colSpan={displayedColumns.length}
+                  role="columnheader"
+                  scope="col"
+                  tabIndex={isGridContext ? -1 : undefined}
+                >
+
+                  <SectionHeader
+                    className={cx('subsection')}
+                    text={subsection.text}
+                    isTitleSticky
+                    boundedWidth={boundingWidth}
+                    level={3}
+                  />
+
+                </th>
+              </tr>
+            </tbody>
+            <tbody className={cx('subsection-rows', {
+              collapsed: isCollapsed,
+              collapsible: isCollapsible,
```

Comment:
Subsections are not collapsible.  We should not need classes on the subsection. The subsection visibility should be covered by the parent section.  We shouldn't even render subsections if the section is collapsed.

Reply:
Subsections don't get rendered, as confirmed by this test. But I can remove these classes if they are redundant.

https://github.com/cerner/terra-framework/blob/92705576279e10198dbac9708a6ee55c676814cc/packages/terra-table/tests/wdio/table-spec.js#L193-L197

Reply:
Removed redundant css here: 0fc369b63